museum/sampleRecords/adminDonor.destination.json

```diff
@@ -0,0 +1,1025 @@
+{
+  "ok":true,
+  "results":[
+    {
+      "object_id":"1",
+      "id":"1",
+      "idno":"A1988.135",
+      "display_label":"STRAWBERRY HILL FARM, ALBANY",
+      "ca_objects.AccessionPrefix":"A",
```

Comment:
I don't think these fields (`AccessionPrefix`, `AccessionYear`, `AccessionNumber`) are required as they are components of the `idno` field so they represent a denormalisation (duplicate data).